README.md

```diff
@@ -1 +1,39 @@
-# sealink-ecom-engine-css
+# Sealink Ecom Engine CSS
+
+SeaLink Ecom Engine shared styles
+
+
+### Usage
+
+    gem install sealink-ecom-engine-css --save
+
+
+### Import the style file to your project main scss/sass
+
+    @import "sealink-ecom-engine-css/sass/ecom"
```

Comment:
Should we rename the stylessheet to something like `ecom-engine-styles`?

Reply:
changed the file name